src/WebSdk/Worker/Sdk/Sdk.props

```diff
@@ -23,6 +24,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
 
   <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+  <Import Condition="'$(WorkerSdkImportPublishSdk)' == 'true'" Sdk="Microsoft.NET.Sdk.Publish" Project="Sdk.props" />
```

Comment:
Today this is getting added in Visual Studio through the publish process. Once this change goes in, we should be able to remove that: 
`<Project Sdk="Microsoft.NET.Sdk.Worker;Microsoft.NET.Sdk.Publish">`

Reply:
Thank you - I completely agree. Think kind of specific, location feedback is exactly what I was hoping you'd bring, and as always you delivered :)

Reply:
done, thanks!